allensdk/brain_observatory/ecephys/stimulus_analysis/__main__.py

```diff
@@ -72,12 +79,25 @@ def add_metrics_to_units_table(nwb_path, stimulus_classes, args):
 
     print(nwb_path)
 
-    session = EcephysSession.from_nwb_path(nwb_path)
+    try:
+        session = EcephysSession.from_nwb_path(nwb_path)
+    except FileNotFoundError:
+        return pd.DataFrame()
```

Comment:
I don't think this is the right behavior. If the input path cannot be found, the module should fail and report the reason for said failure.